src/debugging/coreclr/LocalAspNetCoreSslManager.ts

```diff
@@ -0,0 +1,157 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import * as os from 'os';
+import * as path from 'path';
+import { ConfigurationTarget, MessageItem, workspace, WorkspaceConfiguration } from 'vscode';
+import { DialogResponses } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { PlatformOS } from '../../utils/platform';
+import { quickPickProjectFileItem } from '../../utils/quick-pick-file';
+import { quickPickWorkspaceFolder } from '../../utils/quickPickWorkspaceFolder';
+import { ProcessProvider } from './ChildProcessProvider';
+import { DotNetClient } from './CommandLineDotNetClient';
+import { OSProvider } from './LocalOSProvider';
+import { NetCoreProjectProvider } from './netCoreProjectProvider';
+
+export type HostSecretsFolders = {

```

Comment:
Do we need these to be separate?

Reply:
I don't see the advantage. TS does type-guarding by structure, so passing around variables of types `HostSecretsFolders` and `ContainerSecretsFodlers` interchangeably will not throw any warnings (they are structurally the same, names don't matter). If you want separate types for your own clarifications, it makes sense. But if you are planning to do different operations on varaibles of either type, TS won't prevent the next dev from mixing them up.